src/app/components/action-button/index.html

```diff
@@ -31,4 +31,162 @@
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-inside-long-dialogue-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-section-heading">Workflow placement and behaviors</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use a set of action buttons inside a modal when no page already exists, or the remaining bulk of the task after the action button decision takes place inside a modal. This is common when creating a new record that only requires a standard modal during different versions of its 'Add' process.
+            </p>
+            <ul>
+                <li>Use a 'default' size modal for 2 action buttons and a 'largeModal' for 3 or more.</li>
+                <ul>
+                    <li>Center the action buttons inside the modal, wrapping to another row if needed.</li>
+                </ul>
+                <li>When the user selects a choice, dismiss this modal and open whatever container is appropriate for the remainder of the task.</li>
+                <ul>
+                    <li>If the user cancels or abandons the task afterwards, do not return the user to this modal.</li>
+                </ul>
+                <li>Always provide a "Cancel" action inside a modal with action buttons.</li>
+            </ul>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-in-modal.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            When the user is completing an Add or configuration process contained inside a page or tab (that includes no other task), use action buttons directly on the page. Most 'Add' processes will use the modal example above.
+            </p>
+            <ul>
+                <li>If necessary, include a question above the action buttons to explain the decision being made.</li>
+                <li>Center the question and action buttons inside the page.</li>
+                <li>When the user selects a choice, remove the action buttons and replace them with the consequent next steps of the workflow.</li>
+                <li>If the user leaves the page without selecting a choice, the record or process should be treated as "incomplete" and have any appropriate status indicators on its parent view.</li>
+                <ul>
+                    <li>An explicit "Cancel" or "Delete" action underneath the action buttons and be used if it is valuable for the user to delete the record entirely from this point.</li>
```

Comment:
I think I just have a typo in the original, using "and be used" instead of the intended "can be used". Your suggestion is better anyways.

We should probably rework this to describe only the configuration case. We want to include it in addition to other "back" behaviors. You're very perceptive about the contradictory circumstances of the record status, and you're right.

Reply:
I worked on this some more, but I'm not exactly sure what to say about the configuration case. What do you think of changing this to:

"If it is useful to include an option for users to back out of configuration processes, place an explicit "Cancel" or "Delete" action under the action buttons."